docs/radicle.md

```diff
@@ -15,9 +13,19 @@ version control so powerful in the first place.
 Read more about our [vision for decentralized code
 collaboration](https://radicle.xyz/blog/towards-decentralized-code-collaboration.html).
 
-Radicle also features an [*opt-in* Ethereum integration][https://radicle.xyz/blog/integrating-with-ethereum.html] for
+Radicle also features an [*opt-in* Ethereum integration](https://radicle.xyz/blog/integrating-with-ethereum.html) for
 unique global names, decentralized organizations, and protocols that help maintainers sustain their open-source work.
```

Comment:
I think it would be better to mention Radicle Drips here and link to https://www.drips.network.
Because most of the stuff in the blog post is outdated or being phased out.

Reply:
Good call, and thanks for the tip—hard to tell what's the truest and latest at times.